doc/whatsnew/fragments/1127.false_positive

```diff
@@ -0,0 +1,5 @@
+Fix ``no-name`` false positive in cases when property decorators are used.
```

Comment:
It feels as if this entry should just be about the config. Any other issues should be fixed in a follow up.